client/signup/steps/initial-intent/index.tsx

```diff
@@ -70,6 +70,7 @@ export default function InitialIntentStep( props: Props ) {
 
 	const skipNextNavigation = ( _questionKey: string, _answerKeys: string[] ) => {
 		return (
+			_answerKeys.includes( 'client' ) ||
 			Boolean( getRedirectForAnswers( _answerKeys ) ) ||
 			shouldExitOnSkip( _questionKey, _answerKeys )
```

Comment:
Ideally I'd love to not have to duplicate the logic between `handleNext` and `skipNextNavigation`.
I'll iterate over it in another PR.